wled00/data/index.css

```diff
@@ -461,7 +461,7 @@ button {
 
 .filter {
 	z-index: 1;
-	overflow: hidden;
+	overflow: visible;
```

Comment:
I am not sure this change is appropriate as we do not want filter bar to wrap into two rows.

Reply:
Yeah I didn't this it was a good idea, but saw nothing break, alternative is to add another wrapper and change its size to be tall enough for the tooltips maybe but might have to then account for the top bar option toggle

Reply:
![Screenshot 2023-10-14 at 18 36 24](https://github.com/Aircoookie/WLED/assets/7722392/7efeb432-43d8-4433-aa90-10808e376435)
Not good.